osc-server/src/main/java/org/osc/core/broker/service/vc/DeleteVirtualizationConnectorService.java

```diff
@@ -65,16 +58,4 @@ void validate(EntityManager em, BaseIdRequest request) throws Exception {
         VirtualizationConnectorEntityMgr.validateCanBeDeleted(em, vc);
```

Comment:
@arvindn05 we first validate if the VC can be deleted. I think it is safe to delete VC without the write lock.

Reply:
while we are validating it maybe safe, but when deleting it might not be. We should always write lock and hold it until the vc is safely deleted